### PR TITLE
added a recipe to kill that unwanted input[type=file] standard look.

### PR DESCRIPTION
**Pull request by ntreadway**

### CODE REVIEW

File: file-field-mask/file-field-mask.scss [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    input[type=file]{
+      height:35px;
+      font-size:20px;
+      position: relative;
+      text-align: left;
+      -ms-filter: "progid:DXImageTransform.Microsoft.Alpha(Opacity=0)";
+      filter: progid:DXImageTransform.Microsoft.Alpha(Opacity=0);
+      opacity: 0;
+      z-index: 2;}
+      #fake-file-field {
```
Comment (sxtxixtxcxh) [inline file-field-mask/file-field-mask.scss]: the indentation here is a bit weird.

**Comment by ntreadway** [comment]
updated the nesting.

**Comment by chriseppstein** [comment]
merge it yourself :P I gave you commit

**Comment by ntreadway** [comment]
will do thanks :)

**Pull request closed without merging**